app/views/user_mailer/account_activation.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1><%= t ".html_header" %></h1>
+
+<p><%= t ".hi" %> <%= @user.name %>,</p>
+
+<p>
+<%= t ".text_welcome" %>
+</p>
+
+<%= link_to t(".link_activate"), edit_account_activation_url(@user.activation_token,
+                                                    email: @user.email) %>
```

Comment:
indent nè chú, cho nó đi xa vậy (yaoming)

chị thay url thành path được ko em

Reply:
dạ kooo, vì cái này là nội dung mail gửi cái link đi để activate, nếu gửi path thì nó chỉ gửi cái relative path đi thôi, tức là /abc/xyz, còn để url để gửi toàn bộ link abc.xyz/def/pqr luôn chị :dancing_men: